.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,32 @@
+# Source: https://docs.github.com/en/github/finding-security-vulnerabilities-and-errors-in-your-code/configuring-code-scanning
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ dev, master, github-actions ]
+  pull_request:
+    branches: [ dev ]
+  schedule:
+    - cron: '17 9 * * 6'

```

Comment:
If this is correct: 
![image](https://user-images.githubusercontent.com/55100084/106558429-b3b7df80-64f1-11eb-8d8c-30b1685b9bd4.png)

You want this to run at 9:17 every Sunday..,? Is the idea here that vulnerabilities might update over time, so we need to recheck regularly? Assuming I've understood correctly, I feel like pushes/PRs from normal dev activity would be regular enough to catch updates when those get checked.

Reply:
I believe I just copied and pasted most of the things. I just checked with another repository, and it seems like the time is randomly chosen. I'll get rid of `cron` if we do decide to add this.